bin/install_pepys.ps1

```diff
@@ -9,7 +9,7 @@ $WshShell = New-Object -comObject WScript.Shell
 # Get the User's Send To folder location
 # This is safer than using a hard-coded PATH as network/user settings may mean the folder
 # is in an unexpected place
-$sendto_location = $WshShell.SpecialFolders("SendTo")
+$sendto_location = $WshShell.SpecialFolders("SendTo2")
```

Comment:
@robintw  - I briefly googled, but couldn't any reference for `SendTo2` folder name.  Can you include a URL to some online reference in the previous comment?

Reply:
Aah, glad you caught that - it's a mistake. I'll fix it.